internal/handlers/short.go

```diff
@@ -51,6 +51,8 @@ func parseShortURL(r *http.Request) (*imageDownloadParams, int, error) {
 		params.imageType = "minimal-iso"
 	case "full.iso":
 		params.imageType = "full-iso"
+	case "disconnected-interactive.iso":
+		params.imageType = "disconnected-interactive-iso"
```

Comment:
This is the best name we have for this? It's fine if there are no other options, but it feels a bit unwieldy to me.

Reply:
Yeah, naming is hard:) Maybe simply 'disconnected.iso'? @andfasano wdyt?

Reply:
In this case I feel that the `interactive` part plays quite a relevant role, since it's a special ISO meant not only to support the disconnected installation (ie like the regular ABI ISO), but also to spin up a local agent TUI and moreover a complete assisted UI to support the interactive installation process by the user. Anyhow I don't want to block on this, if no other better ideas will come up I'm fine with `disconnected.iso`

Reply:
That's right, `OVE` is only the current scope, so it will be better to avoid using it in the naming for a more generic product

Reply:
In the context of assisted-service (where users will encounter this), all installs are interactive

- `disconnected-iso`
- `self-contained-iso`
- `jumbo-iso`

not gonna use the 'a' word :speak_no_evil:

Reply:
I like just removing interactive.

+1 for `disconnected-iso`/`disconnected.iso` from me